src/school/Zavod.java

```diff
@@ -0,0 +1,82 @@
+
+package school;
+
+import java.util.Scanner;
+
+public class Zavod {
+    private Scanner sc =new Scanner(System.in);
+    
+    //data
+    private String name;
+    private Zavodnik[] competitors;
+    private int nCompetitors = 0;
+    
+    //konstruktor
+    public Zavod(String name){
+        this.name = name;
+        this.competitors = new Zavodnik[10];
+    }
+    
+    //gettery
+    public String getName() {
+        return name;
+    }
+
+    public Zavodnik[] getCompetitors() {
+        return competitors;
```

Comment:
It is better to return a defensive copy - a new array with a copy of each Zavodnik. For this, you can create a new constructor Zavodnik(Zavodnik z) that creates a new Zavodnik with the same attributes values as z.